ray-operator/controllers/ray/utils/constant.go

```diff
@@ -84,6 +85,13 @@ const (
 	// The default name for kuberay operator
 	ComponentName = "kuberay-operator"
 
+	// The default RayService Identifier.
+	RayServiceCreatorLabelValue = "rayservice"
+
+	// The default suffix for Headless Service for multi-host worker groups.
+	// The full name will be of the form "${RayCluster_Name}-tpu-worker-svc".
+	HeadlessServiceSuffix = "tpu-worker-svc"
```

Comment:
The service may not be limited to TPUs in the future. Perhaps we should consider using a different name.

Reply:
Done, changed to "headless-worker-svc".

Reply:
Oh sorry, I changed the label name but not the suffix value.